### DISCUSSION TITLE
Historical RTM Load Zone

### DISCUSSION THREAD

**Discussion by rammohan4u** [ercot/api-specs#52] [Q&A]
Hello All,

I am new to all this and trying to use REST API to access the ERCOT data. I was researching on this page [https://developer.ercot.com/applications/pubapi/user-guide/openapi-documentation/](url) I am specifically looking for EMIL ID        NP6-785-ER data.Please let me know if there other ways to get this data.

**Comment by CaffeineLab** [comment]
If you aren't comfortable using the API, you can download all of the zip files... could get a bit cumbersome if you are trying to go back more than a few days though.   Otherwise you could try one of the paid services, but that gets' expensive.  How far back are you planning on going?

For the zip files:
https://www.ercot.com/mp/data-products/data-product-details?id=NP6-785-ER

**Reply by rammohan4u** [reply]
Thank you for your response.I already have the manual data,sorry for the confusion. I need to get API working as it is for an application but can't seem to find that particular EMIL in the documentation.I only need it from start of this year and going forward.I need to fetch daily.

**Reply by CaffeineLab** [reply]
Gotcha - I should have tested the endpoint before commenting.

'{"timestamp":"2024-09-25 02:56:31","code":404,"status":"NOT_FOUND","message":"The requested EMIL Product \\"np6-785-er\\" does not seem to be available at the moment, or does not exist."}'

Worst case scenario - grab an html parser like BeautifulSoup and grab the first docid on the page and that's the file you fetch.  Until it becomes available in the API..... 

`<td><a href="https://www.ercot.com/misdownload/servlets/mirDownload?doclookupId=1038115260">zip</a></td>`
